app/views/play/cinematic/CinematicLankBoss.js

```diff
@@ -1,28 +1,63 @@
 import anime from 'animejs/lib/anime.es.js'
-import Promise from 'bluebird'
-import AbstractCommand, { Noop } from './Command/AbstractCommand'
+import { Noop, AnimeCommand } from './Command/AbstractCommand'
+import { getLeftCharacterThangTypeSlug, getRightCharacterThangTypeSlug } from '../../../schemas/selectors/cinematic'
 
 const Lank = require('lib/surface/Lank')
 
-Promise.config({
```

Comment:
Why are we removing this

Reply:
We don't need the cancellable promise, as it was being used in AnimeCommand.